app/models/person.rb

```diff
@@ -898,14 +898,24 @@ def set_ridp_for_paper_application(session_var)
     user.ridp_by_paper_application
   end
 
+  def attributes_changed?
+    person_changes = meaningful_changes(changed_attributes)
+    embedded_changes = embedded_attributes_changed?
+
+    person_changes || embedded_changes
```

Comment:
Could we remove these temp variables and use 
```ruby
meaningful_changes(changed_attributes) || embedded_attributes_changed?
```
Also, what do you think about adding documentation here too? And maybe specs?